SparkPost/2web2ui
# FE-1349 - Removes Codecov integration (for now!)

[FE-1349](https://sparkpost.atlassian.net/browse/FE-1349)

### What Changed

- Removes code instrumentation and code coverage generation from Jest tests and Cypress tests
- Uninstalls relevant packages (`@cypress/code-coverage` and `babel-plugin-istanbul`)
- Reduces test runner parallelism as a result of faster integration tests to save some $$
- Changes strategy for caching dependencies to be more specific - just realized our `restore_cache` and `save_cache` steps were taking a _really_ long time and that was due to [Cypress binary snowballing](https://glebbahmutov.com/blog/do-not-let-cypress-cache-snowball/) - ping me if there are questions about what this change means
- Upgrades Cypress to version `6.3.0` which includes a fix for Electron that prevents that browser from crashing - removed the `--browser` flag from test commands as a result

### How To Test

- Verify that no code coverage is generated in CI
- Verify no code coverage reports are uploaded in CI
- Verify all references to code coverage reporting in the repo are removed
- The CircleCI pipeline should be a _lot_ faster now:
  + Pipeline duration on this branch: **12m 58s**
  + Pipeline duration on `master`: **21m 59s**

### To Do
